SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -238,7 +238,7 @@ SpellDetails = 1
 # Показывать координаты (x,y,z) курсора мыши на Карте Приключений
 # Es zeigt die Koordinaten des Mauszeigers (x,y,z) auf der Abenteuerkarte
 # Δείχνει τις συντεταγμένες του κέρσορα του ποντικιού (x,y,z) στον χάρτη περιπέτειας (Adventure Map)
-AdventureCoordinates = 1
+AdventureCoordinates = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.